src/script/script.cpp

```diff
@@ -267,3 +267,16 @@ std::string CScriptWitness::ToString() const
     }
     return ret + ")";
 }
+
+bool CScript::HasValidOps() const
+{
+    CScript::const_iterator it = begin();
+    while (it < end()) {
+        opcodetype opcode;
+        std::vector<unsigned char> item;
+        if (!GetOp(it, opcode, item) || opcode > 0xb9 || item.size() > MAX_SCRIPT_ELEMENT_SIZE) {
```

Comment:
Let's define a constant for 0xb9, this is kind of a magic number here. `OP_NOP10` would work, but maybe it's better to explicitly mark what the maximum valid opcode is.